setup.py

```diff
@@ -45,8 +39,6 @@
 
 setup(
     version=version,
-    long_description=long_description,
```

Comment:
is long_description no longer something that is used then? I thought it was how you got the readme onto the PyPI page?

Reply:
@timj This is handled by the `file:` directive in the `setup.cfg` file for code-less import of the README into the package metadata.

Reply:
I don't see `file` in `setup.cfg`...